app/views/smart_answers/question.html.erb

```diff
@@ -13,7 +13,11 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <%= render 'smart_answers/debug' %>
-    <%= form_tag current_question_path(@presenter), :method => :get do %>
+    <%= form_tag current_question_path(@presenter), :method => :get, :data => {
+      module: "track-responses",
+      type: question.partial_template_name,
+      "question-key": question.title
+    } do %>
```

Comment:
Again - I think this mix of single and multi-line presentation makes the statement harder to read as on first glance it looks like `module`, `type` and `question-key` are parameters of the parent `form_tag` statement rather than belonging to the inner `data` object. If the statement is totally multi-line the hierarchy of parameters becomes clearer:
```
<%= form_tag(
      current_question_path(@presenter), 
      method: :get, 
      data: {
        module: "track-responses",
        type: question.partial_template_name,
        "question-key": question.title
      } 
    ) do 
%>
```